js/src/frontend-rs/src/lib.rs

```diff
@@ -62,9 +67,9 @@ enum JsparagusError {
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn run_jsparagus(text: *const u8, text_len: usize) -> JsparagusResult {
+pub unsafe extern "C" fn run_jsparagus(text: *const u8, text_len: usize, options: *const JsparagusCompileOptions) -> JsparagusResult {
```

Comment:
I think you should be able to make the input type `&JsparagusCompileOptions`, just a bit nicer, and the output in frontend-rs.h will be the same.

Reply:
Had to land it. Feel free to fix in a followup.